server/aws-lsp-identity/src/language-server/identityService.ts

```diff
@@ -94,7 +108,7 @@ export class IdentityService {
                     clientName: this.clientName,
                     clientRegistration,
                     ssoSession,
-                    handlers: this.handlers,
+                    handlers: this.handlers as Pick<Handlers, keyof SsoHandlers>,
```

Comment:
I thought you made a separate `IamFlowParams` type?  Why is the `Pick` needed here?  Shouldn't these just be `SsoHandlers` in `SsoFlowParams`?